package.json

```diff
@@ -31,10 +21,21 @@
     "eslint-plugin-node": "11.1.0"
   },
   "engines": {
-    "node": ">=10"
+    "node": ">=10",
+    "npm": ">=7"
   },
   "peerDependencies": {
```

Comment:
Maybe we could be less strict with peer dependencies and use "^" on each to give more flexibility to developers. 

Cf https://nodejs.org/uk/blog/npm/peer-dependencies/
> One piece of advice: peer dependency requirements, unlike those for regular dependencies, should be lenient. You should not lock your peer dependencies down to specific patch versions. It would be really annoying if one Chai plugin peer-depended on Chai 1.4.1, while another depended on Chai 1.5.0, simply because the authors were lazy and didn't spend the time figuring out the actual minimum version of Chai they are compatible with.

Reply:
(not sure about the "^" syntax vs ">= 1.5.2 < 2")

Reply:
Good catch, thanks!